index.js

```diff
@@ -73,14 +70,34 @@ function fastifyExpress (fastify, options, next) {
     next()
   }
 
+  /**
```

Comment:
main logic change.

- runs on express only routes
- if the fastify server has a route only run if opt in config is present